internal/services/gcp_reservation_service.go

```diff
@@ -104,7 +104,7 @@ func CreateGCPReservation(w http.ResponseWriter, r *http.Request) {
 
 	launchJob := worker.Job{
 		Type: jobs.TypeLaunchInstanceGcp,
-		Args: &jobs.LaunchInstanceGCPTaskArgs{
+		Args: jobs.LaunchInstanceGCPTaskArgs{
 			AccountID:     accountId,
 			ReservationID: reservation.ID,
 			Zone:          reservation.Detail.Zone,
```

Comment:
Both GCP and AWS are affected, obviously, we have tests for Noop job only so this is where it was correct.